### DISCUSSION TITLE
Tabs instead of a scroll bar in the Loot table editor

### DISCUSSION THREAD

**Discussion by MaxfieldHutton** [IoeCmcomc/MCDatapacker#13] [Ideas]
I havent used much of this program yet but I was having trouble trying to find different pools in the loot table editor until I saw that the scroll bar was the solution. my suggestion would either be to add tabs or maybe some sort of file tree looking thing?
something like this VVV

▼ pools
    ├ 1
    └ 2
    
    again i dont really know much about this program yet but I think thats a pretty good suggestion.
    
    while we are on this menu I'd also like to suggest being able to change the UI font and scale, I have terrible eyesight so I was having trouble reading some of the stuff lol

**Comment by IoeCmcomc** [comment]
Thanks for your suggestions. The design of the loot table editor and similar editors is subject to change. Tabs and list views would be perfect solutions if loot table pools had unique names or IDs. However, since loot table pools are sequential, maybe there are better solutions.
For example, MS Access uses this to navigate between records: 
![image](https://github.com/IoeCmcomc/MCDatapacker/assets/53734763/c14eeff7-cbb8-4bbf-80f2-249a031dd1fb)

I made the scroll bar hiding to give more space on square monitors. I'm going to make it more flexible.

I'll consider adding UI font and scale settings. At least you can already change the font size of the code editor in the settings.
